service/dap/server.go

```diff
@@ -97,26 +97,26 @@ var defaultArgs = launchAttachArgs{
 
 // DefaultLoadConfig controls how variables are loaded from the target's memory,
 // borrowing the default value from the original vscode-go debug adapter and rpc server.
-// For evaluate requests made in the context of 'repl' or 'variables', they are triggered
-// by explicit human action so we use InteractiveEvaluateLoadConfig to apply larger limits.
 // TODO(polina): Support setting config via launch/attach args or only rely on on-demand loading?
-var (
-	DefaultLoadConfig = proc.LoadConfig{
-		FollowPointers:     true,
-		MaxVariableRecurse: 1,
-		MaxStringLen:       64,
-		MaxArrayValues:     64,
-		MaxStructFields:    -1,
-	}
+var DefaultLoadConfig = proc.LoadConfig{
+	FollowPointers:     true,
+	MaxVariableRecurse: 1,
+	MaxStringLen:       64,
+	MaxArrayValues:     64,
+	MaxStructFields:    -1,
+}
 
-	InteractiveEvaluateLoadConfig = proc.LoadConfig{
-		FollowPointers:     true,
-		MaxVariableRecurse: 64,
-		MaxStringLen:       4096,
-		MaxArrayValues:     4096,
-		MaxStructFields:    -1,
-	}
-)
+// For evaluate requests made in the context of 'repl' or 'variables', they are triggered
```

Comment:
Grammar is off here. Sorry for being nitpicky, but every time I read this comment, it throws me off. Should be "Evaluate requests made in the context of 'repl' or 'variables are triggered" by (drop "For" and ", they")